aws/aws.go

```diff
@@ -150,5 +150,11 @@ func newInstance(inst *ec2.Instance) *types.Instance {
 		CloudType: types.AWS,
 		Tags:      tags,
 		Owner:     tags[context.AwsOwnerLabel],
+		Region:    *getRegionFromAvailabilityZone(inst.Placement.AvailabilityZone),
 	}
 }
+
+func getRegionFromAvailabilityZone(az *string) *string {
```

Comment:
Do we need to return a pointer here? We could return simple string with a default value of ""